ProtocolHandler.pm

```diff
@@ -10,6 +10,22 @@ use Slim::Utils::Log;
 use Slim::Utils::Misc;
 use Slim::Utils::Prefs;
 
+use List::Util qw(min max first);
+use Slim::Utils::Errno;
+
+# Size of the _sysread chunksize AND buffers, NOT to be confuesed with same applied to pipeline and/or player.
+
+use constant MAX_OUT    => 512*1024*1024; # safety limit, the out should always be empty, if reached _sysread stops reading from source.
```

Comment:
512MB? You're serious? Please make this feature optional (opt-in). This could easily kill LMS on some systems. Check the dbhighmem preference: if it's not set then don't use your code. It means the user likely doesn't have much memory.

Reply:
that's just a security limit that now is not even settled in LMS. In  normal conditions out buf (before pipeline) is always empty, but today in error conditions could rise with no limit at all, taht away we have a top, that is same value at the largest buffer we could have on players. To me, we could also delete it.